jestProcess.js

```diff
@@ -0,0 +1,10 @@
+const babelOptions = {
+  presets: ["env", "stage-0", "react"],
+  plugins: [
```

Comment:
What are these for? I ran your tests locally without any plugins and it still worked.

I suggest removing this if they are not needed.